plugins/services/src/js/reducers/serviceForm/MultiContainerHealthChecks.js

```diff
@@ -3,79 +3,423 @@ import {
   REMOVE_ITEM,
   SET
 } from '../../../../../../src/js/constants/TransactionTypes';
-import Util from '../../../../../../src/js/utils/Util';
+import {COMMAND, HTTP, HTTPS, TCP} from '../../constants/HealtCheckProtocols';
+import Transaction from '../../../../../../src/js/structs/Transaction';
+import ValidatorUtil from '../../../../../../src/js/utils/ValidatorUtil';
 
-function mapHealthChecks(item) {
-  const newItem = Util.omit(item, ['command', 'protocol']);
+function intOrNull(value) {
```

Comment:
Any chance we can change this function to only "replace" `NaN` with `null`?  It kinda feels wrong to have a function that parses the input into an integer if the input isn't an empty string esp. as you can have strings (e.g. "e2") that still result in `NaN`.